### DISCUSSION TITLE
npm version

### DISCUSSION THREAD

**Discussion by bouteillerAlan** [rpearce/react-medium-image-zoom#284] [Q&A] [answered]
Hello, 

this day i want to use our package so i do `yarn add react-medium-image-zoom`, in my package i have now `"react-medium-image-zoom": "^3.1.3"` but in github i see the package version to `4.3.3`.

Why the npm package is on the old version ?

**Comment by rpearce** [comment, marked as answer]
What in the world... I have no idea. I'll correct this tonight and will check back here. Thank you!

<img width="1261" alt="image" src="https://user-images.githubusercontent.com/592876/122124262-0937bd80-cdfd-11eb-950a-9f04ff7a53fd.png">

**Reply by rpearce** [reply]
Ah, this is because I published a version for `3.x`, but apparently that got marked as the latest version. Oops.

**Reply by rpearce** [reply]
This should now be fixed: 
<img width="897" alt="image" src="https://user-images.githubusercontent.com/592876/122155967-8b54d000-ce57-11eb-94bc-cb736a3edc39.png">
